### ISSUE TITLE
project admin backend

### ISSUE THREAD

**Issue by jonturneratduke** [Duke-Translational-Bioinformatics/duke-data-service#80]
create new project, set status active/inactive, change name

**Comment by dmlond** [comment]
we either need to divide this into 2 cards (Backend, Frontend) or add checkboxes and unassign it.

**Comment by jonturneratduke** [comment]
Yes, good catch. I had it as a task but I saw you were working on 81 so I didn't want to change it while you're in there.
- jon TODO: split out #80/#81 (project admin) to front end and backend cards

Either way would work, checkboxes or subdividing.


I subdivided #67/#68 (user management / tests) earlier this morning, and will do #82/ #83 in a moment.


-

Jon



Jonathan Turner jonathan.turner@duke.edu<mailto:jonathan.turner@duke.edu>
Business Analyst @ Duke School of Medicine, Office of Research Informatics
919.681.2476  / 2424 Erwin Road, Rm 9076 Hock Bldg Durham, NC 27705
